charts/zalenium/values.yaml

```diff
@@ -147,11 +147,15 @@ ingress:
   tls: false
   # secretName: my-tls-cert # only needed if tls above is true
   hostname: zalenium.foobar.com
+  # context path without the forward slash (/)
+  # leaving this empty would run zalenium in the root context (/)
+  contextpath: /
```

Comment:
Same remark: Can you please use the same name `path` as the Kubernetes Ingress object.